CHANGELOG.asciidoc

```diff
@@ -3,6 +3,164 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.2.1]]
+=== Beats version 8.2.1
+https://github.com/elastic/beats/compare/v8.2.0...v8.2.1[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+
+*Auditbeat*
+
+
+*Filebeat*
+
+*Heartbeat*
+
+- Restrict setuid to containerized environments. {pull}30869[30869]
```

Comment:
@emilioalvap according to the PR this is not a breaking change. I can move it to bug fixes, correct?